doc/source/installation.rst

```diff
@@ -138,7 +138,7 @@ Testing your installation
 
     * Through a Singularity container::
      
-        nextflow run VIB-PSB/ksrates --config ./config_elaeis.txt -with-singularity docker://vibpsb/ksrates:latest
+        nextflow run VIB-PSB/ksrates --config ./config_elaeis.txt -with-singularity vibpsb/ksrates:latest
```

Comment:
I tried without docker:// and it could pull the container.
I also found in NF [docs](https://www.nextflow.io/docs/latest/singularity.html#singularity-docker-hub) (the second Note paragraph) that "docker://" is not needed anymore when pulling a Singularity container from Docker Hub.
This is actually nice and we could just use "container = 'vibpsb/ksrates:latest'" in the NF config file, instead of having two different lines.
![image](https://user-images.githubusercontent.com/57489957/129178054-a48921d3-c364-4dd8-9f3f-5c9a477e8659.png)

Reply:
Great! Yes, let's just a single `container = 'vibpsb/ksrates:latest'`!